We don't need this copy command on Linux (or other than Visual Studio
build).  Since it is colliding each other on parallel build, drop the
command for now.

See https://bugs.gentoo.org/666904
diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index 1516775..2deb764 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -109,8 +109,6 @@ foreach(DICT ${DICTS})
       ${DICT}.ocd
     COMMENT
       "Building ${DICT}.ocd"
-    COMMAND
-      ${CMAKE_COMMAND} -E copy "$<TARGET_FILE:libopencc>" "$<TARGET_FILE_DIR:${OPENCC_DICT_BIN}>"
     COMMAND
       ${OPENCC_DICT_BIN}
         --input ${DICT_${DICT}_INPUT}
